kubejs/data/emendatusenigmatica/loot_tables/blocks/dimensional_cluster.json

```diff
@@ -0,0 +1,61 @@
+{
+  "type": "minecraft:block",
```

Comment:
I would much rather see these loot tables modified through scripts. I've been trying to get rid of data pack loot tables where possible: 
`\kubejs\server_scripts\base\loot_tables\blocks`

Reply:
Sure 👍